File: w6/4-cash/cash.py
from cs50 import get_float

change = 0
while change <= 0:
    change = get_float("Change owed: ")

quarters = int(change * 100 / 25)
change = change * 100 % 25

dimes = int(change / 10)
change = change % 10

nickels = int(change / 5)
change = change % 5

pennies = int(change / 1)

# print ("quarters:   " + str(quarters))
# print ("dimes:      " + str(dimes))
# print ("nickels:    " + str(nickels))
# print ("pennies:    " + str(pennies))
# print ("total:      " + str(quarters + dimes + nickels + pennies))
print(quarters + dimes + nickels + pennies)

File: w6/4-cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int change = 0;
    while (change == 0 || change < 0)
    {
        change = get_int("Change owed: ");
    }

    int quarters = change / 25;
    change = change % 25;

    int dimes = change / 10;
    change = change % 10;

    int nickels = change / 5;
    change = change % 5;

    int pennies = change / 1;

    // printf ("quarters:  %i\n", quarters);
    // printf ("dimes:     %i\n", dimes);
    // printf ("nickels:   %i\n", nickels);
    // printf ("pennies:   %i\n", pennies);
    // printf ("total:   %i\n", quarters + dimes + nickels + pennies);
    printf("%i\n", quarters + dimes + nickels + pennies);
}
